slsReceiverSoftware/src/BinaryDataFile.h

```diff
@@ -14,12 +14,10 @@ class BinaryDataFile : private virtual slsDetectorDefs, public File {
 
     fileFormat GetFileFormat() const override;
     void CloseFile() override;
-    void CreateFirstBinaryDataFile(const std::string fPath,
-                                   const std::string fNamePrefix,
+    void CreateFirstBinaryDataFile(const std::string fNamePrefix,
                                    const uint64_t fIndex,
                                    const bool ovEnable,
-                                   const bool sMode, const int modulePos,
-                                   const int nUnitsPerReadout,
+                                   const bool sMode,
```

Comment:
Fun fact: when passing by value (and not reference) the const on parameters in the function declarations doesn't do anything. Only the implementation matters. In principle we could remove the const from const bool in .h and only keep it in the .cpp file.